scripts/triggers-end-to-end-tests/functions/package-lock.json

```diff
@@ -0,0 +1,1927 @@
+{
```

Comment:
We might want to not check in `package-lock.json` files in these end to end tests.  Seems like churn for not much benefit (whereas the root one is more useful)